### PR TITLE
Release Notes - Gift card support multi-currencies

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

Release Notes - Gift card support multi-currencies (EDU-10020)

### CODE REVIEW

File: docs/release-notes/gift-card-support-multi-currencies.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [Create GiftCard in GiftCard Provider](https://developers.vtex.com/docs/api-reference/giftcard-hub-api#post-/giftcardproviders/-giftCardProviderID-/giftcards)
+- [Get GiftCard from GiftCard Provider by ID](https://developers.vtex.com/docs/api-reference/giftcard-hub-api#get-/giftcardproviders/-giftCardProviderID-/giftcards/-giftCardID-)
+
+**GiftCard Provider Protocol API**
+- [Create GiftCard](https://developers.vtex.com/docs/api-reference/giftcard-provider-protocol#post-/giftcards)
+- [Get GiftCard by ID](https://developers.vtex.com/docs/api-reference/giftcard-provider-protocol#get-/giftcards/-giftCardID-)
+
+
+
+>⚠️ Setting a specific currency for each gift card is optional. If you don't want to indicate in which currency it should be created, the behavior will remain the same as in a store using only one currency type, where discounts on the amount are applied without currency conversion.
```
Comment (karenkrieger) [inline docs/release-notes/gift-card-support-multi-currencies.md:27]:
```suggestion
>⚠️ Setting a specific currency for each gift card is optional. If you do not want to indicate in which currency it should be created, the behavior will remain the same as in a store using only one currency type, where discounts on the amount are applied without currency conversion.
```

File: docs/release-notes/gift-card-support-multi-currencies.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+createdAt: 2023-07-27T12:00:00.000Z
+hidden: false
+type: "added"
+excerpt: "Configure your store gift cards in multiple currencies."
+---
+As of July 30, 2023, you will be able to create gift cards in currencies other than those added to your store. This new feature allows you to indicate in which currency the gift card should be created when adding a new gift card in the VTEX Admin.
+ 
+Also, a new optional field (`currencyCode`) has been added to the endpoints below, enabling you to identify the currency applicable to each gift card.
+
+**GiftCard API**
```
Comment (karenkrieger) [inline docs/release-notes/gift-card-support-multi-currencies.md:13]: Por que a API é `GiftCard`e quando se fala da funcionalidade é `gift card`?

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: docs/release-notes/gift-card-support-multi-currencies.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/release-notes/gift-card-support-multi-currencies.md]: Corrigido.

File: docs/release-notes/gift-card-support-multi-currencies.md (hunk 2 above)
Comment (ricardoaerobr) [reply docs/release-notes/gift-card-support-multi-currencies.md:13]: A Clara Cluz (localization) explicou que quando for nome de API, é para manter junto (GiftCard).
Thread: https://vtex.slack.com/archives/C052CEU8GDU/p1689276373065159?thread_ts=1689273609.240369&cid=C052CEU8GDU

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**